cypress/components/button-bar/button-bar.test.tsx

```diff
@@ -2,24 +2,27 @@ import React from "react";
 import {
   Default as ButtonBarCustom,
   DefaultWithWrapper as ButtonBarWithWrapper,
-} from "./button-bar-test.stories";
+} from "../../../src/components/button-bar/button-bar-test.stories";
 import {
   BUTTON_BAR_SIZES,
   BUTTON_BAR_ICON_POSITIONS,
-} from "./button-bar.config";
+} from "../../../src/components/button-bar/button-bar.config";
 
-import { buttonDataComponent } from "../../../cypress/locators/button";
-import { icon } from "../../../cypress/locators";
-import { useJQueryCssValueAndAssert } from "../../../cypress/support/component-helper/common-steps";
-import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import { buttonDataComponent } from "../../locators/button";
+import { icon } from "../../locators";
+import { useJQueryCssValueAndAssert } from "../../support/component-helper/common-steps";
+import CypressMountWithProviders from "../../support/component-helper/cypress-mount";
 
 context("Test for Button-Bar component", () => {
   describe("check props for Button-Bar component", () => {
     it.each([
       [BUTTON_BAR_SIZES[0], 32],
       [BUTTON_BAR_SIZES[1], 40],
       [BUTTON_BAR_SIZES[2], 48],
-    ])("should set size to %s for a Button-Bar", (size, px) => {
+      // https://github.com/bahmutov/cypress-each/issues/2
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+    ])("should set size to %s for a Button-Bar", (size: string, px: number) => {
```

Comment:
```suggestion
    ])("should set size to %s for a Button-Bar", (size, px) => {
```

You can remove the `ts-ignore` and lint disable comment

Reply:
Unfortunately we need to instruct the compiler to ignore this line, due to the bug with the `cypress-each` package's types which @DlgSHi has commented on line 22. Without disabling the compiler we get a type error